content/articles/ssl-certificates-email-validation.markdown

```diff
@@ -0,0 +1,163 @@
+---
+title: SSL Certificate Email-based Domain Validation
+excerpt: The email-based domain validation is the most common domain ownership validation method for a certificate and it is required for domain-validated certificates.
+categories:
+- SSL Certificates
+redirect_from:
+  - /articles/ssl-certificates-email-approval/
+  - /articles/selecting-ssl-certificates-email/
+  - /articles/resending-ssl-certificates-email/
+  - /articles/changing-ssl-certificates-email/
+---
+
+# SSL Certificate Email-based Domain Validation
+
+### Table of Contents {#toc}
+
+* TOC
+{:toc}
+
+---
+
+**Email-based certificate validation** is the most common certificate validation mechanism used by Certificate Authorities in case of [domain-validated certificate orders](/articles/ssl-certificates-types/#ssl-certificates-by-validation-level).
+
+The goal of validation is to ensure the authenticity of a certificate order before issuing a new certificate. Specifically, before issuing the certificate, the Certificate Authority must be sure the domain listed in the certificate is registered and someone with admin rights is aware of and approves the certificate request.
+
+The validation email is a mandatory step performed by the Certification Authority based only on publicly accessible information.
+
+## The process
+
+The email validation process consists of three steps:
+
+1. When you purchase a certificate, we show you a list of authoritative emails identified by the Certificate Authority for the domain associated with the certificate
+1. You select an email from the list.
+1. The [Certificate Authority](/articles/what-is-certificate-authority/) sends a verification email (also called DCV email) to the recipient with an unique link to approve the certificate and validate your domain ownership.
+1. **You click on the link to validate and approve the certificate**. At this point, the certificate's authenticity is validated and the authority will generate it.

```

Comment:
The use of section-steps to me denotes a list of real steps the user is expected to follow in the UI. Generally, it maps side-by-side instructions.

In this case, it's a list of generic activities, in order, that you are supposed to do. But there is no specific UI correspondence.